src/Components/Aspire.Azure.Npgsql/AssemblyInfo.cs

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire;
+using Aspire.Npgsql;
+using Azure.Core;
+
+[assembly: ConfigurationSchema("Aspire:Npgsql", typeof(NpgsqlSettings))]
+[assembly: ConfigurationSchema("Aspire:Npgsql:Credential", typeof(TokenCredential))]
```

Comment:
@eerhardt is that correct?

Reply:
```suggestion
```

Reply:
We can probably get rid of this whole thing since it references `Aspire.Npgsql` underneath, which will bring in this configuration schema.